pom.xml

```diff
@@ -31,11 +31,11 @@
 
     <properties>
         <connect-runtime-version>2.0.0</connect-runtime-version>
-        <confluent.version>5.4.0</confluent.version>
+        <confluent.version>5.4.1</confluent.version>
```

Comment:
```suggestion
```

We can (and should) remove this line; it's [already defined](https://github.com/confluentinc/common/blob/f668cc36070d6fdcbde9b7c9854e84d302c38346/pom.xml#L50) in the common parent pom.

Reply:
Thanks @C0urante , addressed